docs/topics/Permissions.md

```diff
@@ -30,57 +30,63 @@ Below is a table of all current permissions, their integer values in hexadecimal
+| CREATE_GUILD_EXPRESSIONS               | `0x0000080000000000` `(1 << 43)` | Allows for creating emojis, stickers, and soundboard sounds, and editing and deleting those created by the current user. Not yet available to developers, [see changelog](#DOCS_CHANGE_LOG/clarification-on-permission-splits-for-expressions-and-events). |                            |
+| CREATE_EVENTS                          | `0x0000100000000000` `(1 << 44)` | Allows for creating scheduled events, and editing and deleting those created by the current user. Not yet available to developers, [see changelog](#DOCS_CHANGE_LOG/clarification-on-permission-splits-for-expressions-and-events).                        | V, S                       |
+| USE_EXTERNAL_SOUNDS                    | `0x0000200000000000` `(1 << 45)` | Allows the usage of custom soundboard sounds from other servers                                                                                                                                                                                            | V                          |
+| SEND_VOICE_MESSAGES                    | `0x0000400000000000` `(1 << 46)` | Allows sending voice messages                                                                                                                                                                                                                              | T, V, S                    |
+| SEND_POLLS                             | `0x0002000000000000` `(1 << 49)` | Allows sending polls                                                                                                                                                                                                                                       | T, V, S                    |
+| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows user-installed apps to send public responses. When disabled, users will still be allowed to use their apps but the responses will be ephemeral. This only applies to apps not also installed to the server.                                         | T, V, S                    |
+
+| Channel Type (Abbreviated) | Description | Channel Type                                                                                    |
+|----------------------------|-------------|-------------------------------------------------------------------------------------------------|
+| T                          | Text        | GUILD_TEXT, GUILD_ANNOUNCEMENT, ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD, GUILD_FORUM |
+| V                          | Voice       | GUILD_VOICE                                                                                     |
+| S                          | Stage       | GUILD_STAGE_VOICE                                                                               |
```

Comment:
not sure if it makes sense to include the thread types here, since threads don't have permission overwrites (they just inherit from their parent channels).

also should include `GUILD_MEDIA` under text, its basically the same as a forum

```suggestion
| Channel Type (Abbreviated) | Description | Channel Types                                            |
|----------------------------|-------------|----------------------------------------------------------|
| T                          | Text        | GUILD_TEXT, GUILD_ANNOUNCEMENT, GUILD_FORUM, GUILD_MEDIA |
| V                          | Voice       | GUILD_VOICE                                              |
| S                          | Stage       | GUILD_STAGE_VOICE                                        |
```

Reply:
Makes sense! Thanks for the suggested change